tests/core/test_entities.py

```diff
@@ -184,52 +184,6 @@ def test_set_input_with_constructor():
     assert_near(person('salary', "2017-10"), [2000, 3000, 1600, 0, 0])
 
 
-def test_person_without_household():
```

Comment:
Sorry ; nothing. It was about unknown entity but I see now that it was tested earlier.